UnitTestBot/UTBotJava
# Update of module `utbot-python-types`

## Description

- Implemented custom JsonAdapter for much faster parsing of structures with mypy information
- Updated mypy from `1.0.0` to `1.5.1`: some behavior changed, so a few bugs had to be fixed (maybe we will find more).
- Small fixes in `utbot-python` module.

## How to test

### Automated tests

Unit tests of module `utbot-python-types`.

### Manual tests

Run UTBot on `langchain` project. Loading of type information should have become significantly faster.

## Self-check list

Check off the item if the statement is true. Hint: [x] is a marked item.

Please do not delete the list or its items.
